src/utils/pgwire/src/pg_response.rs

```diff
@@ -58,7 +62,7 @@ impl std::fmt::Display for StatementType {
 pub struct PgResponse {
     stmt_type: StatementType,
     row_cnt: i32,
-
+    notice: Option<String>,
```

Comment:
There are perhaps multiple lines of notice.

```
postgres=# drop schema t;
ERROR:  cannot drop schema t because other objects depend on it
DETAIL:  table t.t depends on schema t
table t.test depends on schema t
HINT:  Use DROP ... CASCADE to drop the dependent objects too.
```

I'm not sure if "\n" works to split the lines in a single String. If not I would suggest using `Vec<String>`

```
notice_lines: Vec<String>
```